### DISCUSSION TITLE
How to set voltage range in app

### DISCUSSION THREAD

**Discussion by ntik98** [fhdm-dev/scoppy#154] [Scoppy Front end designs]
Hi everyone,

I make circuit using single LM324. When applying sine wave (1khz) with amplitude 5v , in app only show 3.3 volt.

So what change should I make in app to show correct value.

**Comment by fhdm-dev** [comment]
Tap the connection badge at the lower left of the screen and then tap 'Connected device' and then tap 'Firmware settings' and then tap 'Channels'. From there you can select the channel and configure the voltage range(s).
